### DISCUSSION TITLE
Adding size in jsx is not working...

### DISCUSSION THREAD

**Discussion by AbhayaShankar** [ChartsCSS/charts.css#78] [Q&A] [answered]
I have imported the `import "charts.css";`
and yet after when I use `style = "--size: calc(xyz)"` , Its not working

I am passing dynamic sizes in the size using calc but the norm is not supporting
style cannot be defined using "--size" kindof structure

**Comment by kiley0** [comment, marked as answer]
If you are using React with TypeScript, try using the `style` attribute with bracket notation, like this:

```
<td style={{ ["--size" as any]: "calc(60 / 100)" }}>60</td>
```

**Reply by AbhayaShankar** [reply]
Ok Great! Will try it out...
